src/doc/en/faq/faq-general.rst

```diff
@@ -335,6 +335,9 @@ Here is a BibTeX entry for Sage:
 
 Adjust version/year as needed.  You might also like to use DOI for Sage,
 as the note entry in the above record, or directly as DOI record.
+The DOI 10.5281/zenodo.8042260 represents all Zenodo DOIs for Sage;
+clicking on :doi:`10.5281/zenodo.8042260` will show DOI for the latest
+Sage version, presently 10.3, see :doi:`10.5281/zenodo.10841614`.
 
```

Comment:
It would be better not to hard-wire "the present version" of sage.

Reply:
> It would be better not to hard-wire "the present version" of sage.

this is meant to show that clicking on "stable" DOI brings up the latest one, so the URL gets rewritten.